### PR TITLE
Allow local rules

### PR DESCRIPTION
**Pull request by MrSeccubus**
### Description

Allow the user to write custom rules in local.cf

### Issues Resolved

I wanted to add custom rules to local.cf, but could not

### Contribution Check List

- [ ] All tests pass. 
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README and metadata if applicable.

See [CONTRIBUTING.md](https://github.com/zuazo/spamassassin-cookbook/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -53,3 +53,4 @@
 ]
 default['spamassassin']['spamd']['pidfile'] = '/var/run/spamd.pid'
 default['spamassassin']['spamd']['nice'] = nil
+default['spamassassin']['conf']['localrules'] = []
```
Comment (zuazo) [inline attributes/default.rb:56]: Could you add an example to the readme about how to use it?

File: metadata.rb [hunk 2]
```diff
@@ -26,7 +26,7 @@
 Installs and configures SpamAssassin, a mail filter software to identify spam.
 EOH
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '2.1.0' # WiP
+version '2.1.0001' # WiP
```
Comment (zuazo) [inline metadata.rb]: Please, do not change the version.

**Review by zuazo — CHANGES_REQUESTED** [review]
Thanks for the PR :smile: I have added some comments to your contribution.

File: metadata.rb (hunk 2 above)
Comment (MrSeccubus) [reply metadata.rb]: The reason for the changed version number is to give the local copy of the cookbook priority over the community version pending the mergen request, bit I should have excluded it for the PR.

File: attributes/default.rb (hunk 1 above)
Comment (MrSeccubus) [reply attributes/default.rb:56]: Will do.

**Comment by MrSeccubus** [comment]
I believe that commit fa4d17e should cover all comments

File: attributes/default.rb (hunk 1 above)
Comment (MrSeccubus) [reply attributes/default.rb:56]: Done

File: metadata.rb (hunk 2 above)
Comment (MrSeccubus) [reply metadata.rb]: Done

**Comment by MrSeccubus** [comment]
Back to you, @zuazo

**Review by zuazo — APPROVED** [review]
Oh! You even added some test. Fantastic work man! :+1:

**Pull request merged**